database/staking.go

```diff
@@ -8,6 +8,7 @@ import (
 	stakingtypes "github.com/cosmos/cosmos-sdk/x/staking/types"
 	dbtypes "github.com/forbole/bdjuno/database/types"
 	"github.com/forbole/bdjuno/x/staking/types"
+	bstaking "github.com/forbole/bdjuno/x/staking/types"
```

Comment:
I think we can remove this import as it's the same as the one in the line above, and use `types.` instead of the currenct `bstaking.`